express-app/controllers/user.js

```diff
@@ -3,6 +3,7 @@ const req = require('request');
 const isOnline = require('is-online');
 const urls = require('../routes/urls');
 const Estudio = require('../models/estudio');
+const Section = require('./section');
```

Comment:
This is a bit confusing, the name looks like a model, you should change it to 
`sectionController`  to keep consistency in the name of the models and controllers.